src/heretic/model.py

```diff
@@ -93,8 +104,12 @@ def reload_model(self):
             self.settings.model,
             dtype=dtype,
             device_map=self.settings.device_map,
+            trust_remote_code=self.trusted_models.get(self.settings.model),
```

Comment:
Not sure about this. If the user explicitly passes `--trust-remote-code True`, we'd expect this to also extend to the evaluated model, no?